iocage/lib/Config/Jail/BaseConfig.py

```diff
@@ -68,13 +68,14 @@ class BaseConfig(dict):
         'iocage.lib.Config.Jail.'
         'JailConfigProperties.JailConfigProperties'
     )
-    data: dict = {}
+    data: dict
 
     def __init__(
         self,
         logger: 'iocage.lib.Logger.Logger'=None
     ) -> None:
 
+        self.data = {}
```

Comment:
what's the difference here?

Reply:
Previously it was a class variable, now it's an instance variable. Makes a difference when working with multiple jails (like we do when using templates)